app/src/main/java/org/mozilla/fenix/crashes/CrashReporterFragment.kt

```diff
@@ -8,12 +8,15 @@ import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.widget.TextView
 import androidx.fragment.app.Fragment
 import androidx.navigation.Navigation
 import kotlinx.android.synthetic.main.fragment_crash_reporter.*
 import mozilla.components.lib.crash.Crash
+import org.jetbrains.anko.find
```

Comment:
We're using anko for A/B testing layouts. It may be reasonable to use anko here, too. However, the Android framework finally became smart enough to auto-cast findViewById() calls in the past couple years. The convenience here is probably a wash and not really worth it anymore.

Reply:
Whoops! Meant to use `findViewById` here :)